docs/on-your-servers.rst

```diff
@@ -0,0 +1,20 @@
+Running these on your Servers
+=============================
+
+
+To run these on your own Servers, you need to create and store 2 repositories of private data. These repositories should not be public.
```

Comment:
This page is missing important context - the 'why'. How about "It's possible to deploy the code for any OCP hosted service (such as Kingfisher, Pelican or the Data Review Tool) on your own infrastructure - for integration into your own web services, for example. This page describes the steps necessary to set up your own version of this repository and associated configuration"

Reply:
Ok